pavelib/utils/test/bokchoy_utils.py

```diff
@@ -92,6 +94,14 @@ def wait_for_server(server, port):
             if int(response.status) == 200:
                 server_ok = True
                 break
+
+        # TODO: in python 3.3, socket.error is deprecated in favor of ConnectionRefusedError
+        except socket_error as s_error:
+            # We expect this error until the server comes up
+            if s_error.errno == errno.ECONNREFUSED and attempts == 0 :
+                # Give it a few more seconds, it takes
+                # a bit for the server to come up initially
+                time.sleep(10)
```

Comment:
Sure that sounds good. I'll bump it to 30 for now. I think that in the future we will implement some solution wherein everything is containerized and services register that they're up.